abci/Cargo.toml

```diff
@@ -37,7 +37,7 @@ tracing-subscriber = { version = "0.3", optional = true, default-features = fals
     "ansi",
     "env-filter",
 ] }
-thiserror = "1.0.39"
+thiserror = { version = "1.0.39" }
```

Comment:
why this change is needed

Reply:
I'm unsure how it happened that our codestyle became like that, but when nothing except `version` is specified it's only a string a rest is omitted; so in case you want to unify it I'd suggest to make the rest right

Reply:
well I meant the opposite